functions_lab.js

```diff
@@ -0,0 +1,40 @@
+//Question 1
+const findAverage = (num1, num2) => {
+    let sum = num1 + num2
+    let average = sum/ 2
+   return average
+    }
+   console.log(findAverage(4, 6));
+
+//Question 2
+const avOfThree = (num1, num2, num3) => {
+    let sum = num1 + num2 + num3
+    let average = sum / 3
+    return average
+   }
+   console.log(avOfThree(1, 3, 5));
+
+   //Question 3
+let arr = []
```

Comment:
You want to declare array inside the function so its reusable, having it outside the function will make it so if you use it again the result will be added to the previous time it was used...